File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main() {
    ios::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    
    int a, b, c;
    cin >> a >> b >> c;

    if (a+b+c != 180)
        cout << "Error" << "\n";
    else {
        if (a == b && b == c)
            cout << "Equilateral" << "\n";
        else if (a == b || b == c || c == a)
            cout << "Isosceles" << "\n";
        else
            cout << "Scalene" << "\n";
    }
    
    return 0;
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
import sys

angle = []

for _ in range(3):
    angle.append(int(sys.stdin.readline()))

if sum(angle) != 180:
    sys.stdout.write("Error")
else:
    if angle[0] == angle[1] == angle[2]:
        sys.stdout.write("Equilateral")
    elif angle[0] != angle[1] and angle[0] != angle[2] and angle[1] != angle[2]:
        sys.stdout.write("Scalene")
    else:
        sys.stdout.write("Isosceles")